gulpfile.js

```diff
@@ -107,7 +108,7 @@ function setEnvironment(cb) {
 }
 
 // Define complex tasks
-const build = gulp.series(setEnvironment, clean, copy, gulp.parallel(css, js, img));
+const build = gulp.series(setEnvironment, clean, copy, gulp.parallel(css, js, img), versionAssets);
```

Comment:
It would be more performant to version assets in the gulp stream. This way we won't need to do these tasks twice:
- Glob file search
- File content read